packages/mds-geography/api.ts

```diff
@@ -1,104 +1,48 @@
-      }
-
-      /* If the user can only read published geos, and all they want is the unpublished metadata,
-       * throw a permissions error.
-       */
-      try {
-        if (!scopes.includes('geographies:read:unpublished') && params.get_unpublished) {
-          throw new InsufficientPermissionsError(
-            'Cannot require unpublished geo metadata without geography:read:unpublished scope'
-          )
-        }
-
-        /* If the user has only the read:published scope, they should not be allowed to see
-         * unpublished geos. If they didn't supply any params, we modify them here so as to
-         * filter only for published geo metadata. We have to monkey with the params here
-         * in a way that we don't for the bulk read of the geographies since we can't filter
-         * the DB results in this layer, since metadata has no idea if the geo it's associated
-         * with is published or not.
-         */
-        if (
-          !scopes.includes('geographies:read:unpublished') &&
-          params.get_unpublished === null &&
-          params.get_published === null
-        ) {
-          params.get_published = true
-        }
-        const geography_metadata = await db.readBulkGeographyMetadata(params)
-        return res.status(200).send({ version: res.locals.version, geography_metadata })
-      } catch (error) {
-        logger.error('failed to read geography metadata', error)
-        /* This error is thrown if both get_published and get_unpublished are set.
-         * To get all geos, neither parameter should be set.
-         */
-        if (error instanceof BadParamsError) {
-          return res.status(400).send({ error })
-        }
-        if (error instanceof InsufficientPermissionsError) {
-          return res.status(403).send({ error })
-        }
-        return res.status(500).send({ error })
-      }
-    }
-  )
 
   app.get(
     pathsFor('/geographies/:geography_id'),
     checkGeographyApiAccess(scopes => {
       return scopes.includes('geographies:read:published') || scopes.includes('geographies:read:unpublished')
     }),
-    async (req: GeographyApiRequest, res: GetGeographyResponse) => {
+    async (req: GeographyApiRequest, res: GetGeographyResponse, next: express.NextFunction) => {
       const { geography_id } = req.params
       try {
         const geography = await db.readSingleGeography(geography_id)
         if (!geography.publish_date && !res.locals.scopes.includes('geographies:read:unpublished')) {
           throw new InsufficientPermissionsError('permission to read unpublished geographies missing')
         }
         return res.status(200).send({ version: res.locals.version, geography })
-      } catch (err) {
-        logger.error('failed to read geography', err.stack)
-        if (err instanceof NotFoundError) {
-          return res.status(404).send({ error: err })
+      } catch (error) {
+        logger.error('failed to read geography', error.stack)
```

Comment:
Can get rid of this logging line, we only care about logging the error in the case of a ServerError (which is handled by the global middleware now).